index.html

```diff
@@ -0,0 +1,166 @@
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Twitter Challenge</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!-- LEFT COLUMN -->
+    <section class="left">
+      <p class="logo">
+        <img src="/images/book.png" alt="" width="35" height="35" />
+      </p>
+      <div>
+        <div class="navbar">
+          <ul class="nav-icons">
+            <li>
+              <img src="/images/birdhouse.png" alt="" height="35" width="35" />
+            </li>
+            <li>
+              <img src="/images/hashtags.png" alt="" height="35" width="35" />
+            </li>
+            <li>
+              <img src="/images/bell.png" alt="" height="35" width="35" />
+            </li>
+            <li>
+              <img src="/images/mail.png" alt="" height="35" width="35" />
+            </li>
+            <li>
+              <img src="/images/open-book.png" alt="" height="35" width="35" />
+            </li>
+            <li>
+              <img src="/images/list.png" alt="" height="35" width="35" />
+            </li>
+            <li>
+              <img src="/images/user.png" alt="" height="35" width="35" />
+            </li>
+            <li>
+              <img src="/images/more.png" alt="" height="25" width="25" />
+            </li>
+          </ul>
+          <strong class="nav-buttons"
+            ><p class="homebutt">Home</p>
+            <p>Explore</p>
+            <p>Notifications</p>
+            <p>Messages</p>
+            <p>Bookmarks</p>
+            <p>Lists</p>
+            <p>Profile</p>
+            <p>More</p></strong
+          >
+        </div>
+        <div class="nav-icons"><p class="twtbutton">Tweet</p></div>
+      </div>
+    </section>
+
+    <!-- MAIN COLUMN -->
+
+    <section class="main">
+      <div class="row1">
+        <div class="upper-row">
+          <strong>Home</strong>
+          <img src="/images/favorite.png" height="35" width="35" alt="" />
+        </div>
+        <img class="dp" src="/images/boxer.png" alt="" width="50" height="50">
+        <div class="type-con">
+         
+          <div class="in-type-con">What's happening?</div>
+        </div>
+        <ul class="m-icons">
+          <li>
+            <img src="/images/picicon.png" alt="" height="20" width="20" />
+          </li>
+          <li>
+            <img src="/images/gificon.png" alt="" height="20" width="20" />
+          </li>
+          <li>
+            <img src="/images/bar-chart.png" alt="" height="20" width="20" />
+          </li>
+          <li><img src="/images/smile.png" alt="" height="20" width="20" /></li>
+          <li>
+            <img src="/images/calendar.png" alt="" height="20" width="20" />
+          </li>
+        </ul>
+        <div class="left-side"><p class="twtbutton2">Tweet</p></div>
+      </div>
+
+      <div class="row2">
```

Comment:
It is nice that you named this division as "row2", but... it is a post, right? maybe "post" would be more understandable